engine/src/test/scala/cromwell/webservice/routes/wes/ServiceInfoSpec.scala

```diff
@@ -24,7 +25,7 @@ class ServiceInfoSpec extends AsyncFlatSpec with ScalatestRouteTest with Matcher
 
   val expectedResponse = WesStatusInfoResponse(Map("CWL" -> Set("v1.0"), "WDL" -> Set("draft-2", "1.0", "biscayne")),
     List("1.0"),
-    Set("ftp", "s3", "drs", "gcs", "http"),
+    Set("ftp", "s3", "drs", "gcs", "http", "blob"),
```

Comment:
Hm, does this indicate that we're reporting the blob filesystem to WES users as something they can use? I don't think we want to do that yet.

Reply:
Good point, I'll look into whats happening here